scripts/roundtrip.py

```diff
@@ -6,10 +6,18 @@
 import os
 
 
+class RunError(Exception):
+    pass
+
+
 def run_cmd(cmd, timeout):
     print("Running: ", ' '.join(cmd))
-    return subprocess.run(cmd, capture_output=True,
-                          timeout=timeout, text=True)
+    try:
+        p = subprocess.run(cmd, capture_output=True,
+                           timeout=timeout, text=True)
+    except:
+        raise RunError
```

Comment:
It would be nice if there was some sort of message attached to each instance of `RunError`, which could be printed when handled and that would carry some information about what failed. You can take inspiration from `AssertionError`.